src/Base/AMReX.cpp

```diff
@@ -0,0 +1,47 @@
+#include <pybind11/pybind11.h>
```

Comment:
Let's include this consistently, otherwise this led to ODR issues as soon as we start to bind some STL classes in one of TUs, IIRC.
```suggestion
#include <pybind11/pybind11.h>
#include <pybind11/stl.h>
```